rules/Proj0400.md

```diff
@@ -0,0 +1,26 @@
+# Proj0400: Define the project publishability explicitly
+To prevent confusion, explicitly define the
+`<IsPublishable>` node.
+
```

Comment:
Add a link to MS documentation to explain what publishable is about, as this has meaning for the rules to come.

Reply:
May be this one: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-publish

Reply:
✔️